rockcraft.yaml

```diff
@@ -93,6 +93,8 @@ parts:
     overlay-script: |
       AWS_JAVA_SDK_BUNDLE_VERSION='1.12.540'
       HADOOP_AWS_VERSION='3.3.6'
+      ICEBERG_SPARK_RUNTIME_VERSION='3.4_2.12'
+      ICEBERG_VERSION='1.4.3'
```

Comment:
**out-of-scope** but worth considering for the future, can we move these constants at the top of the file such that they are easily to be identified (with respect to have them nested in the code), but most importantly, could we enable renovate on these? @welpaolo

Reply:
I like the idea, I need to look in that. I think we can manage it but it will require a little bit of research and tests.

Reply:
Yes, let's leave that for another PR. Can you just create a Jira ticket and post the link here?

Reply:
https://warthogs.atlassian.net/browse/DPE-3446